src/node/nodestate.h

```diff
@@ -1250,14 +1246,14 @@ namespace ccf
       }
     }
 
-    void read_ledger_idx(raft::Index idx)
+    void read_ledger_idx(consensus::Index idx)
     {
-      RINGBUFFER_WRITE_MESSAGE(raft::log_get, to_host, idx);
+      RINGBUFFER_WRITE_MESSAGE(consensus::log_get, to_host, idx);
     }
 
-    void log_truncate(raft::Index idx)
+    void log_truncate(consensus::Index idx)
```

Comment:
While renaming the messages to `log_`, this method should also be `ledger_truncate`

Reply:
done